cypress/utils/storeFront/collectionsUtils.js

```diff
@@ -0,0 +1,24 @@
+import Collections from "../../apiRequests/storeFront/Collections";
+
+export const isCollectionVisible = (collectionId, channelSlug) => {
+  const collectionsRequest = new Collections();
+  return collectionsRequest
+    .getCollection(collectionId, channelSlug)
+    .then(resp => {
+      const collection = resp.body.data.collection;
```

Comment:
These two lines seem to be almost identical in products & collections (and I assume is or will be in some other places), could be moved to a util function